controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -3349,6 +3349,10 @@ bool master_thread::handle_cmdu_control_message(const std::string &src_mac,
         LOG(TRACE) << "ACTION_CONTROL_CHANNEL_SCAN_TRIGGER_SCAN_RESPONSE for mac " << hostap_mac;
         break;
     }
+    case beerocks_message::ACTION_CONTROL_CHANNEL_DUMP_SCAN_RESULTS_RESPONSE: {
+        LOG(TRACE) << "ACTION_CONTROL_CHANNEL_DUMP_SCAN_RESULTS_RESPONSE for mac " << hostap_mac;
+        break;
+    }
```

Comment:
As I wrote: the response isn't used for anything, so there is no need to add it.

There is of course the symmetry of having a request/response for everything. However, the symmetry is already broken a bit through the different notification messages. And for sure, I don't think it warrants adding roughly 100 lines of code just for symmetry.

Reply:
I partly agree, while I do think that adding code for symmetry's sake is not a good reason, I have used the additional logs that the response prints to validate that the request has gone through and has performed correctly so I still see a use for the request/response symmetry, even if the only reason is logging